grafana/provisioning/datasources/default.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: 1
+
+datasources:
+  - name: PostgreSQL
+    type: postgres
+    url: 127.0.0.1:65432
```

Comment:
@hengsokly please update port to use default Postgres port 5432 as 65432 is a custom we used in our case